doc/integrator/c2cwsgiutils.rst

```diff
@@ -0,0 +1,10 @@
+.. _integrator_c2cwsgiutils
+
+c2cwsgiutils
+============
+
+c2cgeoportal use c2cwsgiutils you can need it for sone integrtion task, see his
```

Comment:
"c2cgeoportal" -> "c2cgeoportal uses"
"integrtion" -> "integration"
"his" -> "its"

But I am not sure what we are trying to say in the sentence? What kind of integration task can an integrator need it for?

Reply:
You can see all of them here: https://geomapfish-demo-dc.camptocamp.com/2.4/
short list:
- get packages versions
- changing log level
- memory diff
- enable SQL profiler
- ...

Reply:
So suggestion:
"c2cgeoportal uses c2cwsgiutils. You can use functionalities from c2cwsgiutils to assist in integration tasks. See its documentation."

Reply:
+1 thanks :-)